.werft/values.dev.yaml

```diff
@@ -6,11 +6,11 @@ installation:
   shortname: "dev"
 hostname: staging.gitpod-dev.com
 imagePrefix: eu.gcr.io/gitpod-core-dev/build/
-certificatesSecret: {}
-  # secretName: proxy-config-certificates
-  # fullChainName: tls.crt
-  # chainName: tls.crt
-  # keyName: tls.key
+certificatesSecret:
```

Comment:
when deploying without HTTPS, this secret should not be present.

Reply:
Outch, fixed.